src/dataQuery.ts

```diff
@@ -75,10 +75,11 @@ function walkPath(filters: Filters, p: ViewPath): Filters {
 
 export function sanitizeFilter(
   filter: Filter,
-  tag: `#${string}`
+  tag: `#${string}`,
+  returnEmptyFilter?: boolean
```

Comment:
There should be no case where we query an empty filter. Empty Filter = give me all events

Reply:
bad naming - it's just an empt tags-array, but nonetheless, it's not necessary to have at all